lib/Options.php

```diff
@@ -40,10 +43,16 @@ class Options {
 
     private $shutdownTimeout = 3000; // milliseconds
 
+    private $dynamic_cache;
```

Comment:
It's intentionally an underscore to distinguish it from the actual options.

Where to put it instead so that it is accessible from deflateResponseFilter()?
And yeah, Options is a value object, but value objects may contain dynamic entities…